src/common/computeRequiresStrictStatusChecksForRefs.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { computeRequiresStrictStatusChecksForRefs as computeRequiresStrictStatusChecksForReferences } from './computeRequiresStrictStatusChecksForRefs';
+
+/**
+ * Tests
+ */
+describe('computeRequiresStrictStatusChecksForRefs', (): void => {
+  it('returns false for all refs when no branch protection rules exist for the repository', (): void => {
+    expect.assertions(1);
+
+    const result = computeRequiresStrictStatusChecksForReferences(
+      [],
+      ['master', 'dev'],
+    );
+
+    expect(result).toStrictEqual([false, false]);
+  });
+
+  it('returns false for all refs when none of the branch protection rule patterns match provided refs', (): void => {
+    expect.assertions(1);
+
+    const result = computeRequiresStrictStatusChecksForReferences(
+      [
+        {
+          pattern: 'test1',
```

Comment:
Should we add a test to check whether micromatch is working?

Reply:
We should indeed. Added